cmd/deployment-check/README.md

```diff
@@ -110,20 +110,26 @@ spec:
   podSpec:
     containers:
       - name: deployment
-        image: kuberhealthy/deployment-check:v1.6.2
-        imagePullPolicy: IfNotPresent
+        image: kuberhealthy/deployment-check:v1.8.0
+        imagePullPolicy: Always
         env:
           - name: CHECK_DEPLOYMENT_REPLICAS
             value: "6"
           - name: CHECK_DEPLOYMENT_ROLLING_UPDATE
             value: "true"
+          - name: TOLERATION_VALUE
```

Comment:
You should leave this out if the value is just an empty string

Reply:
we will be using this env variable. If I remove it completely someone would need to read through the code and add this manually to their spec file to use. My code already looks for this to be empty and assigns a default nil in the event it is. Let me know how you want me to proceed.

Reply:
this is why we have a list of configurable things in the readme

not everyone is going to be using the environment variable -- we use it yes, but we can modify those values in the spots where we use them

Reply:
understand. Thank you